homeassistant/components/media_player/denon.py

```diff
@@ -51,6 +53,8 @@ def __init__(self, name, host):
         self._host = host
         self._pwstate = 'PWSTANDBY'
         self._volume = 0
+        self._source_list = {'TV': 'SITV', 'TUNER': 'SITUNER',
```

Comment:
Since you're using the keys for the normal user representation in the UI, it would make sense to not make them uppercase or use underscores.